packages/bpk-component-datatable/src/bpk-data-table-column.scss

```diff
@@ -33,11 +41,25 @@
   overflow: hidden;
 }
 
-.bpk-data-table-column__sort-icon {
+.bpk-data-table-column__sort-icons {
   margin: 0 0 0 $bpk-spacing-xs;
-  fill: $bpk-color-gray-700;
 
   @include bpk-rtl {
     margin: 0 $bpk-spacing-xs 0 0;
   }
 }
+
+.bpk-data-table-column__sort-icon {
+  fill: $bpk-color-gray-300;
+  display: block;
+  &:hover {
+    fill: $bpk-color-gray-500;
+  }
+  &--selected {
+    &, &:hover { fill: $bpk-color-blue-700 };
+  }
+}
+
+.bpk-data-table-column__sort-icon--up {
+  margin-bottom: -6px;
```

Comment:
I'm not sure if this is the right way of specifying such a custom value.. maybe there's a better css solution out there too.